homeassistant/components/opentherm_gw/sensor.py

```diff
@@ -47,19 +47,30 @@ def __init__(self, gw_dev, var, device_class, unit, friendly_name_format):
         self._device_class = device_class
         self._unit = unit
         self._friendly_name = friendly_name_format.format(gw_dev.name)
+        self._unsub_options = None
         self._unsub_updates = None
+        self._force_update = False
```

Comment:
Same as in `binary_sensor.py`, we should load the setting from the `config_entry` here if it exists.

Reply:
Good catch. I've fixed this in the latest commit and also extracted the default value into a constant in const.py